### PR TITLE
Add --drop-constraint flag

### PR DESCRIPTION
**Pull request by proxeter**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ on:
 
 jobs:
   goreleaser:
+    permissions: write-all
```
Comment (andreynering) [inline .github/workflows/release.yml]: In theory this is not needed?

It already worked without this on both this repo and others.

Comment (proxeter) [reply .github/workflows/release.yml]: In our fork goreleaser job failed with error without this permission

```
  ⨯ release failed after 1m49s               error=scm releases: failed to publish artifacts: PATCH https://api.github.com/repos/viru-tech/testfixtures/releases/77677546: 403 Resource not accessible by integration []
Error: The process '/opt/hostedtoolcache/goreleaser-action/1.11.4/x64/goreleaser' failed with exit code 1
```

This changes made like [this](https://stackoverflow.com/a/70448851)
I can remove this permission or downgrade it to specify only needed scope

Comment (andreynering) [reply .github/workflows/release.yml]: Perhaps it's a different setting between these repositories. I'd prefer that you delete the last commit and, if on release I get the same error, then I can add it back.

Comment (proxeter) [reply .github/workflows/release.yml]: Sure. I've reverted last commit

**Comment by andreynering** [comment]
Thanks @proxeter!

**Pull request merged**